frontend/src/components/assessments/student-experience/question-components/Image.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { Image as Img } from "@chakra-ui/react";
```

Comment:
nitpick:
```suggestion
import { Image as ChakraImage } from "@chakra-ui/react";
```
I think that's the convention we've been following for renamed Chakra imports)